include/SFML/Graphics/Text.hpp

```diff
@@ -299,14 +375,45 @@ class SFML_GRAPHICS_API Text : public Drawable, public Transformable
     void ensureGeometryUpdate() const;
 
     ////////////////////////////////////////////////////////////
+    /// \brief Adds a line to the vertices
+    ///
+    /// This used for adding strikethrough and underline lines
+    ///
+    /// \param position         Width and top of the line
```

Comment:
The documentation is correct. It is the width and top right now as it is implemented and there is a 3rd parameter for the vertical offset. However looking over this is does make sense to change it to "left and top" and change the 3rd parameter to the width of the line... unless I am forgetting something from the last time I looked at this.